File: projects/order-creation-queries/core/src/main/java/com/robinfood/core/entities/menu/MenuSuggestedPortionResponseEntity.java
package com.robinfood.core.entities.menu;

import lombok.Data;

import java.util.List;

@Data
public class MenuSuggestedPortionResponseEntity {

    private final List<MenuSuggestedPortionDataEntity> changes;
    private final Long id;
    private final String image;
    private final String name;
    private final Long parentId;
    private final String sku;
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/entities/menu/MenuSuggestedPortionResponseEntity.kt
package com.robinfood.core.entities.menu

open class MenuSuggestedPortionResponseEntity(
        val changes: List<MenuSuggestedPortionDataEntity>?,
        id: Long?,
        image: String?,
        name: String?,
        parentId: Long?,
        sku: String?
): MenuPortionEntity(null, id, image, name, parentId, null, sku)